plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/gather_nd.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/gather_nd.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> dPrecisions = {
+        InferenceEngine::Precision::FP32,
+        InferenceEngine::Precision::FP16,
+        InferenceEngine::Precision::I32,
+        InferenceEngine::Precision::I64,
+        InferenceEngine::Precision::I16,
+        InferenceEngine::Precision::U8,
+        InferenceEngine::Precision::I8
+};
+const std::vector<InferenceEngine::Precision> iPrecisions = {
+        InferenceEngine::Precision::I32,
+};
+
+const auto gatherNDArgsSubset1 = ::testing::Combine(
+        ::testing::ValuesIn(std::vector<std::vector<size_t>>(
```

Comment:
Please format the` ::testing::Combine` parameter one value per line by adding `//` at the end of each line

Reply:
Ok, I think it is already formated, just too long each line.